client/src/store/actions/serverCommendsActions.js

```diff
@@ -1,12 +1,36 @@
-import { userActions } from './';
+import { editUserInfoMessageType, userJoinedMessageType, userRemovedMessageType, getRandomNameMessageType, otherUserEditedMessageTyped } from '../../common/messageTypes';
```

Comment:
Consider using "import *..." syntax just to shorten line length

Reply:
I was wonderting if that would be ok, felt kind of wrong but ok :)